src/pharmpy/tools/modelsearch/tool.py

```diff
@@ -258,8 +258,12 @@ def post_process(rank_type, cutoff, *models):
         results_to_summarize.extend(model.modelfit_results for model in res_models)
 
     summary_models = summarize_modelfit_results(results_to_summarize)
-    summary_models['step'] = [0] + [1] * (len(results_to_summarize) - 1)
-    summary_models.reset_index().set_index(['step', 'model'])
+    summary_models.insert(
```

Comment:
Could this be replaced with `summary_models['step'] = [0] + [1] * (len(summary_models) - 1)` ? If that works it might be a bit simpler.

Reply:
Oooh, did not think of that. I'll check it out and get back to you

Reply:
Implemented your suggestion :)